### PR TITLE
LPAL-911 Improve cypress scripts

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

[LPAL-911](https://opgtransform.atlassian.net/browse/LPAL-911)

## Approach

See ticket.

## Learning

n/a

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [X] I have added mandatory tags to terraformed resources, where possible
* [X] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [X] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

File: cypress/s3_monitor.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+    def assume_role_and_get_client(self):
+        sts = boto3.client(
+            "sts",
+            region_name="eu-west-1",
+        )
+
+        if self.in_ci:
+            print("S3Monitor starting, Assuming CI role")
+            role_arn = "arn:aws:iam::050256574573:role/opg-lpa-ci"
```
Comment (williamfalconeruk) [inline cypress/s3_monitor.py:31]: ensure this is not hard coded here, as i think you might want to run this on the preproduction also.

File: cypress/s3_monitor.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        :param options: dict with v (verbose) and c (is in CI) boolean properties
+        """
+        self.verbose = options["v"]
+        self.in_ci = options["c"]
+        self.s3Client = self.assume_role_and_get_client()
+
+    def assume_role_and_get_client(self):
+        sts = boto3.client(
+            "sts",
+            region_name="eu-west-1",
```
Comment (williamfalconeruk) [inline cypress/s3_monitor.py:26]: make this a parameter, will help with DR

File: cypress/s3_monitor.py (hunk 1 above)
Comment (townxelliot) [reply cypress/s3_monitor.py:33]: Where would the correct place be that I could get the strings for these roles from? Somewhere in terraform? Note that this is a conditional which switches between roles (and not something I touched), but you're right that it should be fixed.

File: cypress/s3_monitor.py (hunk 2 above)
Comment (townxelliot) [reply cypress/s3_monitor.py:28]: What's the correct value to use from terraform for this? Presumably I can pass an env var over which contains this.

**Pull request closed without merging**